docs/examples/openapi/plugins/custom_plugin.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import Any
+
+from litestar.connection import Request
+from litestar.openapi.plugins import OpenAPIRenderPlugin
+
+
+class RapidocRenderPlugin(OpenAPIRenderPlugin):
+    """Render an OpenAPI schema using Rapidoc."""
+
+    def __init__(
+        self,
+        *,
+        version: str = "9.3.4",
+        js_url: str | None = None,
+        path: str | Sequence[str] = "/rapidoc",
+        **kwargs: Any,
+    ) -> None:
+        """Initialize the OpenAPI UI render plugin.
+
+        Args:
+            version: Rapidoc version to download from the CDN. If js_url is provided, this is ignored.
+            js_url: Download url for the RapiDoc JS bundle. If not provided, the version will be used to construct the
+                url.
+            path: Path to serve the OpenAPI UI at.
+            **kwargs: Additional arguments to pass to the base class.
```

Comment:
do we want all of the docstring stuff in the examples?

Reply:
Happy for you to push in any changes you want to the docs @JacobCoffee

Reply:
removed - also I think I'll change the example case to the scalar plugin since that's the default one.